docs/en/compute/Spark-On-Kubernetes.md

```diff
@@ -160,10 +165,15 @@ $ ./bin/spark-submit --master k8s://https://<kubernetes-api-server>:8443 \
 local:///opt/spark/examples/jars/spark-examples_2.11-2.4.4.jar \
 alluxio://<alluxio-master>:19998/LICENSE
 ```
-> Note: You can find the address of the Kubernetes API server by running `kubectl cluster-info`.
-You can find more details in Spark [documentation](https://spark.apache.org/docs/latest/running-on-kubernetes.html?q=cluster-info#cluster-mode).
-You should also using the properties corresponding to your 
-[domain socket volume type]({{ '/en/compute/Spark-On-Kubernetes.html#short-circuit-operations' | relativize_url }}).
+
+Note:
```

Comment:
Does `>` work with `-` so you can list items in a note? That way you stay consistent for the note format. @Xenorith

Reply:
From what I saw, the `>` syntax didn't render properly into a note on the webpage. I also don't think the list syntax would render as a bullet list if prepended by the `>` syntax like the following:
```
> Note:
> - item 1
> - item 2
```